### ISSUE TITLE
Hide irrelevant data in the repository list

### ISSUE THREAD

**Issue by nipunvashistha** [openSUSE/open-build-service#4686]
In reference to issue #2529 

Issue/Feature description
====================
I think we should change the way repositories are shown. There is a lot of information load for the user in that particular section, which confuses the user and fills up too much space (For instance, if there are 40 repos listed there, then a user will have to scroll all the down to find a particular repo). We could reduce this by hiding some irrelevant info to the user. 

One way to solve this could be by adding accordions:

![group 7](https://user-images.githubusercontent.com/33966661/37618135-3c2def88-2bdb-11e8-9d93-3edaea5a33ac.png)
![group 6](https://user-images.githubusercontent.com/33966661/37618141-3e1cdcdc-2bdb-11e8-9b41-2fbb1af1e674.png)

OBS instance:
https://build.opensuse.org/project/show/Apache

**Comment by coolo** [comment]
why you think the number of build results is irrelevant?

**Comment by adrianschroeter** [comment]
it is not easy to spot build failures this way at least ....

**Comment by nipunvashistha** [comment]
@coolo The number of build result is not irrelevant, I agree. And this might not be the perfect solution, but grouping build results would clean things up, enhance the readability and creating less confusion. And incase when there are suppose 100 build results then it would be better to hide some information.

**Comment by nipunvashistha** [comment]
@adrianschroeter Agreed. Maybe we could mark the builds having issues with some kind of warning symbol in the closed accordion. (so that we could look into them)
In addition. we could stack the builds with issues on top to make it more obvious.

**Comment by nipunvashistha** [comment]
I will be uploading the follow up screenshots soon.

**Comment by Ana06** [comment]
@nipunvashistha 

> The number of build result is not irrelevant, I agree

I think that @coolo just mean the opposite. If the build successes or fails is a really important information, I do not think that hiding it is a good idea. @openSUSE/open-build-service any other opinion?
